docs/agent.md

```diff
@@ -0,0 +1,59 @@
+# agent (Private Beta)
```

Comment:
should `agent` be upper case or not?

Reply:
Haha it bugged me too. I was just copying what was there for the station docs.

I'll capitalize it since it looks better that way. Perhaps the intention was to do something like our hcp-docs where it's `command` but, for the Title I personally think normal text is fine... feel free to push back though